### DISCUSSION TITLE
✅ Toolkit v0.1.1

### DISCUSSION THREAD

**Discussion by flanakin** [microsoft/finops-toolkit#45] [Releases]
### 🎯 Primary goals
PowerShell commands to help automate the deployment, cleanup, and management of [FinOps hubs](https://aka.ms/finops/hubs), Cost Management exports, and to convert cost data to the FinOps Open Cost and Usage Specification (FOCUS).

### 🆕 New in 0.1.1
- 🖥️ [FinOps toolkit PowerShell commands](https://aka.ms/ftk/ps):
  - Convert cost data to the FinOps Open Cost and Usage Specification (FOCUS).
  - Get and delete Cost Management exports
- 🌐 [Open data files](https://microsoft.github.io/finops-toolkit/data)
  - PowerShell commands to get open data sets for pricing units, regions, and services.
- 🏦 [FinOps hubs](https://aka.ms/finops/hubs)
  - PowerShell commands to get and delete FinOps hubs.

### ℹ️ Updated
- 📒 [Cost optimization workbook](https://microsoft.github.io/finops-toolkit/optimization-workbook) updated to cover more scenarios.

### 🔜 Pushed to next release
- 🖥️ [PowerShell commands](https://aka.ms/ftk/ps)
  - Add and remove FinOps hubs scopes
  - Create Cost Management exports

> [!NOTE]
> _Templates are not yet available in the Azure Quickstart Templates repository yet. Please see downloads below!

### 🙋‍♀️ How you can help
1. Are there any PowerShell commands you'd like to see included in this release?
2. Are we missing anything you'd expect in this release?
3. Is there anything you feel is more critical before this release?
4. Thinking about going past v0.1.1, what do you want to see next?

<br>

📋 Milestone tasks: [Toolkit v0.1.1](https://github.com/microsoft/finops-toolkit/milestone/14)
🌱 Feature branches: [powershell](https://github.com/microsoft/finops-toolkit/tree/features/powershell), [focus](https://github.com/microsoft/finops-toolkit/tree/features/focus)

---
### 📥 [Download v0.1.1](/microsoft/finops-toolkit/releases/v0.1.1) &nbsp; &nbsp; &nbsp; 📜 [Changelog](https://aka.ms/ftk/changes#-v011) &nbsp; &nbsp; &nbsp; 📦 [Templates](https://aka.ms/finops/toolkit/templates) &nbsp; &nbsp; &nbsp; 📗 [Learn more](https://aka.ms/finops/toolkit)

**Comment by krazy-glue** [comment]
hello,
I don't really know where to post this but here I try..

Is there a reason the powerBI template needs an account for the Cost management connector event if we use the FINHUB exports?

**Reply by flanakin** [reply]
Sorry, missed this. Hubs 0.1.x was using the connector to pull actual cost since it was built to only rely on a single dataset for the first release. This was removed in 0.2. There is still one reference to the connector to pull reservation recommendations, but this will also be removed in a future release.

**Reply by flanakin** [reply]
You can follow this feature request to track us adding other datasets: https://github.com/microsoft/finops-toolkit/issues/83

If you would like to see us track any one dataset before the others, please create a specific issue for that and we can prioritize that one dataset first.
